### DISCUSSION TITLE
I am a student and was involved in the year long demo of copilot. Now I am unable to access it.

### DISCUSSION THREAD

**Discussion by IsomerX** [community/community#46168] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I am a student and was involved in the year long demo of copilot. Now I am unable to access it.
It just says: GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Comment by SilasRodrigues19** [comment]
Hey @IsomerX 

I found some answered discussions on this topic, maybe it can help you:

#43695
#44054
#46045
#43634

**Comment by davecheney** [comment, marked as answer]
@IsomerX if you are a current student, please ensure your student coupon is active on your github account then when you visit https://github.com/github-copilot/signup the student subscription will be available.
